Model/Response/Transactions/Cancel.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Balance Payments For Magento 2
+ * https://www.getbalance.com/
+ *
+ * @category Balance
+ * @package  Balancepay_Balancepay
+ * @author   Developer: Pniel Cohen
+ * @author   Company: Girit-Interactive (https://www.girit-tech.com/)
```

Comment:
what is this @dhiraj-toptal ?

Reply:
I copied old file as its a template for these class, I will remove it from this file.